prime-router/src/test/kotlin/fhirengine/engine/FhirRouterTests.kt

```diff
@@ -102,14 +102,14 @@ data object SampleFilters {
         "%patient.birthDate.exists()",
         "%specimen.type.exists()",
         "(%patient.address.line.exists() or " +
-                "%patient.address.postalCode.exists() or " +
-                "%patient.telecom.exists())",
+            "%patient.address.postalCode.exists() or " +
```

Comment:
Just confirming that this is just formatting?

Reply:
Correct.